cypress/support/commands.js

```diff
@@ -22,4 +22,94 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
```

Comment:
`nextPageByButton`, `nextPageByNav`, and `categorizeColumn` are implemented for clarity in commonly reused test code.

`passIndex` and `passCategorization` remain in use for UI-based (not programmatic state-based) testing.